modules/dynamic-apiserver/pkg/cli/service_unarchive_operation.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"fmt"
+
+	client2 "github.com/go-openapi/runtime/client"
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/pkg/generated/client"
+	"github.com/spf13/viper"
+
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/pkg/generated/client/service"
+
+	"github.com/spf13/cobra"
+)
+
+func makeServiceUnarchiveCmd(apiClientFunc func() (*client.ServiceAPI, error)) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "serviceUnarchive",
+		Short:   `Unarchive service object`,
+		Aliases: []string{"unarchive"},
+		RunE:    runServiceUnarchive(apiClientFunc),
+	}
+
+	_ = cmd.PersistentFlags().String(idFlag, "", "service id")
```

Comment:
1. Mark it as Required (cmd.MarkRequiredFlag)
2. Help text: `Required. Service ID`

Reply:
Added MarkRequiredFlag. Help text fixed.